Dockerfile

```diff
@@ -27,7 +27,7 @@ COPY . .
 RUN npm run build
 # RUN ls -la
 # RUN ls -laR dist/
-CMD ["node", "dist/app.js"]
+CMD ["npm", "run serve:prod"]

```

Comment:
Shouldn't it be `CMD ["npm", "run", "serve:prod"]` instead?

Reply:
This is if is served as a single container...

Reply:
@alv67 as a matter of fact my suggestion was to replace

```text
CMD ["npm", "run serve:prod"]
```

with

```text
CMD ["npm", "run", "serve:prod"]
```

as on my local instance (Ubuntu 20.04.2 LTS 64-bit) the command you propose triggers a runtime error when the container is run

```text
...
server_1        | 
server_1        | Usage: npm <command>
server_1        | 
server_1        | where <command> is one of:
server_1        |     access, adduser, audit, bin, bugs, c, cache, ci, cit,
server_1        |     clean-install, clean-install-test, completion, config,
server_1        |     create, ddp, dedupe, deprecate, dist-tag, docs, doctor,
server_1        |     edit, explore, fund, get, help, help-search, hook, i, init,
server_1        |     install, install-ci-test, install-test, it, link, list, ln,
server_1        |     login, logout, ls, org, outdated, owner, pack, ping, prefix,
server_1        |     profile, prune, publish, rb, rebuild, repo, restart, root,
server_1        |     run, run-script, s, se, search, set, shrinkwrap, star,
server_1        |     stars, start, stop, t, team, test, token, tst, un,
server_1        |     uninstall, unpublish, unstar, up, update, v, version, view,
server_1        |     whoami
server_1        | 
server_1        | npm <command> -h  quick help on <command>
server_1        | npm -l            display full usage info
server_1        | npm help <term>   search for help on <term>
server_1        | npm help npm      involved overview
server_1        | 
server_1        | Specify configs in the ini-formatted file:
server_1        |     /root/.npmrc
server_1        | or on the command line via: npm <command> --key value
server_1        | Config info can be viewed via: npm help config
server_1        | 
server_1        | npm@6.14.13 /usr/local/lib/node_modules/npm
server_1        | 
blobfishes_static-pages_1 exited with code 0
blobfishes_server_1 exited with code 1
gmacario@gmpowerhorse:~/github/aquariophilie/blobfishes (client-integration)$
```